_posts/2022-12-12-async-manual.md

```diff
@@ -1462,17 +1453,17 @@ protected Exception(String message, Throwable cause,
 }
 ```
 
-Just set ```writableStackTrace``` to ```false```. I will also be ugly, but faster.
+Just set ```writableStackTrace``` to ```false```. It will be rather ugly but faster.
 
 ### G1GC humongous allocation
 {: #methods-g1ha }
 
-I've already mentioned how to detect humongous objects with allocation mode. Since an
-async-profiler can also instrument JVM code, and allocation of a humongous object is 
-nothing else than invocation of part of a code in C, we can take advantage of that.
-If you want to check where humongous objects are allocated you can use method mode
+We already saw how to detect humongous objects with allocation mode. Since
+async-profiler can also instrument JVM code, and allocations of a humongous objects are 
+nothing else than invocations of C++ code, we can take advantage of that.
+If you want to check where humongous objects are allocated, you can use method mode
```

Comment:
```suggestion
If you want to check where humongous objects are allocated, you can use native functions mode
```

Reply:
Method mode is not the same as native functions. It's a bit undocumented, but Andrei ask me to separate those two.

Reply:
Good to know